go/ql/test/library-tests/semmle/go/aliases/defsuses/test.ql

```diff
@@ -0,0 +1,37 @@
+import go
+
+newtype TEntityWithDeclInfo = MkEntityWithDeclInfo(Entity e)
+
+class EntityWithDeclInfo extends TEntityWithDeclInfo {
```

Comment:
You could make this file a bit more succinct by having a char pred which stores `e` in a field, so you can use it directly in the member predicates. You can also add a getter for it and use it in the top-level predicates. But this is a purely aesthetic point, so feel free to ignore.